pkg/util/util.go

```diff
@@ -46,6 +46,40 @@ func KubeconfigWriteToFile(filename string, clusterURL string, clusterTrustBundl
 	return nil
 }
 
+// GetExternalIP get the generated external IP from service
+func GetExternalIP(clusterType string) (string, error) {
+	config, err := rest.InClusterConfig()
+	if err != nil {
+		return "", err
+	}
+
+	clientset, err := kubernetes.NewForConfig(config)
+	if err != nil {
+		return "", err
+	}
+
+	ns := GetComponentNamespace()
+	if clusterType == "eks" {
```

Comment:
if you want to discovery the external ip automatically, I think we should find it from service, e.g.

if service type is lb, we will get the host directly
if service type is nodeport, we will get node Internal ip
if service type is clusterip, maybe have ocp route or ingess
if service type is external, get the external host/ip